docs/usage/tables/formats.md

```diff
@@ -41,57 +41,57 @@ The following creates a new table in the `path/to/storage` LOCATION two colummns
 
 === "Parquet"
 
-    ```bash
-    bin/uc table create --full_name unity.default.test \
-       --columns "some_numbers INT, some_letters STRING"  \
-       --storage_location /path/to/storage \ 
-       --format PARQUET
-    ```
+```bash
```

Comment:
This change will break mkdocs rendering. 

Before:
![Screenshot 2024-10-10 at 21 43 16](https://github.com/user-attachments/assets/e9fbb865-2c1c-4d8e-bd48-3cbab2cf6892)

After:
![Screenshot 2024-10-10 at 21 42 29](https://github.com/user-attachments/assets/89a2ef1b-363c-45f0-bdb1-9778cca473b3)